org-code-javabuilder/lib/src/main/java/org/code/javabuilder/LambdaRequestHandler.java

```diff
@@ -105,8 +105,17 @@ public String handleRequest(Map<String, String> lambdaInput, Context context) {
     final AmazonS3 s3Client = AmazonS3ClientBuilder.standard().build();
     final AWSFileManager fileManager =
         new AWSFileManager(s3Client, outputBucketName, javabuilderSessionId, getOutputUrl, context);
+    final LifecycleNotifier lifecycleNotifier = new LifecycleNotifier();
+    // TODO: Should GlobalProtocol.create() happen in CodeExecutionManager#onPreExecute so it can
+    // also be destroyed in #postExecute?
```

Comment:
should this comment still be here? What clean up do we need to do on the protocol instance besides the cleanUpResources we already call?

Reply:
Good catch, I think this was left around from an earlier idea - I was thinking maybe we needed to null out the GlobalProtocol instance when cleaning up, but not sure if that's necessary anymore

Reply:
Reworded this comment, but gonna leave this TODO here because I think we can still move some common setup steps (like creating GlobalProtocol) down into CodeExecutionManager so its shared by lambda and websocketserver.